components/search/ResultsList.js

```diff
@@ -306,7 +306,7 @@ const ResultItem = ({msmt}) => {
                   <ASNBox asn={msmt.probe_asn} />
                 </Box>
                 <Box width={4/16} title={moment.utc(msmt.measurement_start_time).format('YYYY-MM-DD HH:mm [UTC]')}>
-                  {moment.utc(msmt.measurement_start_time).format('YYYY-MM-DD HH:mm')}
+                  {moment.utc(msmt.measurement_start_time).format('YYYY-MM-DD HH:mm [UTC]')}
```

Comment:
Same for this, we should check if the `UTC` string goes inside of square brackets or not.

Reply:
I did check and seems inside bracket works for me. But @sarathms do let me know if you think this is not a good way.

Reply:
1. For consistency,  let us use `intl.formatDate` for places where we display dates.

2. I am not sure we have real estate to add the UTC string to the date column here. It looks pretty cluttered. This is why we made a compromise and added a tooltip which showed the same value but with the UTC suffix.

![image](https://user-images.githubusercontent.com/700829/72929886-b9001d00-3d28-11ea-85f0-f3c09d25b62b.png)

Unless there is a better UX solution, I suggest we revert this show what it showed earlier, but using the `intl.formatDate()` API instead of `moment.format()` for consistency.